### PR TITLE
Switch to GitHub actions on CI

### PR DESCRIPTION
**Pull request by jtpio**
Extracting the GitHub Actions part of #86, so it can be used for other PRs such as #90.

### CODE REVIEW

**Comment by jtpio** [comment]
cc @fcollonval @ianhi since this was discussed in #86.

Example run in the fork: https://github.com/jtpio/widget-ts-cookiecutter/runs/2011603959

For now this PR skips tests and docs, just so we can have a basic workflow working for other PRs. But we should eventually add them back, and also check the generated workflow from the cookiecutter is also correct.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        cd jupyter-widget-testwidgets
+        pip install .
+        jupyter nbextension enable --py --sys-prefix jupyter_widget_testwidgets
+        # Validate nbextension (enable does not use exit code):
+        python -c "from notebook.nbextensions import validate_nbextension; import sys; sys.exit(validate_nbextension('jupyter_widget_testwidgets/extension') or 0)"
+        pip install jupyterlab
+        # Make sure our lab extension was installed.
+        jupyter labextension list 2>&1 | grep -ie "jupyter-widget-testwidgets.*OK"
+        # Make sure our lab extension can be linked.
+        jupyter labextension link
```
Comment (ianhi) [inline .github/workflows/build.yml:84]: One the PR for jlab3 we can update this to check if `jupyter labextension develop . --overwrite` works.

**Review by ianhi — APPROVED** [review]
Looks good thanks!

File: .github/workflows/build.yml (hunk 1 above)
Comment (jtpio) [reply .github/workflows/build.yml:84]: yes absolutely we can do that in https://github.com/jupyter-widgets/widget-ts-cookiecutter/pull/90

**Pull request merged**